clang/lib/Serialization/ASTReader.cpp

```diff
@@ -3912,6 +3912,11 @@ llvm::Error ASTReader::ReadASTBlock(ModuleFile &F,
       FPPragmaOptions.swap(Record);
       break;
 
+    case DECLS_WITH_EFFECTS_TO_VERIFY:
+      for (unsigned I = 0, N = Record.size(); I != N; /*in loop*/)
```

Comment:
```suggestion
      for (unsigned I = 0, N = Record.size(); I != N;)
```

Reply:
Also, does `ReadDeclID` increment `I`?  That seems really frightning/surprising.

Reply:
Iirc this is just mirroring what is done elsewhere in that same file/function.

Reply:
E.g.
https://github.com/llvm/llvm-project/blob/273871a3633ed4fe87f17738d79a91d1ea6ba621/clang/lib/Serialization/ASTReader.cpp#L3933-L3941

Reply:
AH, right, i see the `ReadDeclID` thing now, I forgot that the serializer did that.  I've never noticed the `in loop` thing before, seems a little silly, but as it is elsewhere here *shrug*.